spec/chewy/index/actions_spec.rb

```diff
@@ -680,4 +680,42 @@
   describe '.journal' do
     specify { expect(DummiesIndex.journal).to be_a(Chewy::Journal) }
   end
+
+  describe '.clear_cache' do
+    before do
+      stub_model(:city)
+      stub_index(:cities) do
+        define_type City
+      end
+    end
+
+    context do
+      let(:index_name) { 'test_index' }
+      let(:index_name_with_prefix) { 'cities_test_index' }
+      let(:unexisted_index_name) { 'wrong_index' }
+
+      context 'with existed index' do
```

Comment:
I would add a test to cover using `clear_cache` without arguments once you set its default